flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/PartitionRequestClientFactoryTest.java

```diff
@@ -58,6 +59,38 @@
 
 	private static final int SERVER_PORT = NetUtils.getAvailablePort();
 
+	@Test
+	public void testInterruptsNotCached() throws Exception {
+		ConnectionID connectionId = new ConnectionID(new InetSocketAddress(InetAddress.getLocalHost(), 8080), 0);
```

Comment:
Sorry for the stupid question, but is this actually using the port for network traffic or is it just a dummy value?
If it's actually using, I'd propose to use port 0 (random free port).

Reply:
It's a port netty will to connect to. So it doesn't matter which port to use (as long we aren't interested in any results).
This port is also used throughout the class.

Reply:
I'm more worried that the test will fail if something else is running on that port.

Reply:
The request will likely fail (usually nothing is running on 8080), but the result is ignored, so it won't affect the test.

Reply:
Okay if the test is not affected by that I'll merge.